snippets/curl/get_nb_assets.sh

```diff
@@ -0,0 +1,116 @@
+!/bin/sh
```

Comment:
The shebang is missing the "she" part.

```suggestion
#!/bin/sh
```